packages/samples/visibility/visibility.cadl

```diff
@@ -45,9 +45,21 @@ model WriteableOptionalPerson {
   ...OptionalProperties<WritablePerson>;
 }
 
+model ListResult<T> {
+  items: T[];
+}
+
 @route("/hello")
 namespace Hello {
   @get op read(@path id: string, @query fieldMask: string[]): ReadablePerson;
   @post op create(@body person: WritablePerson): ReadablePerson;
   @put op update(@body person: WriteableOptionalPerson): ReadablePerson;
+
+  @Cadl.Rest.listsResource(Person)
+  @get
+  op list(): ListResult<Person>;
+
+  @put
```

Comment:
Hmm, missed this before, but why isn't this patch? I expected it to only work for patch.